src/NHSD.BuyingCatalogue.Identity.Api/Certificates/Certificate.cs

```diff
@@ -58,35 +58,16 @@ private static X509Certificate2 Load(ILogger logger, CertificateSettings setting
                 throw new CertificateSettingsException("Error with certificate or settings", e);
             }
         }
-        private static byte[] GetBytesFromPem(string pemString, PemStringType type)
+        private static byte[] GetBytesFromPem(string pemString, string sectionToken)
```

Comment:
It might be worth adding guards to the parameters to ensure they are not null, especially `pemString` as this looks like data from external source.

Reply:
It's all wrapped in a try... catch block, which will handle & output a decent error message for all the different kinds of invalid input. I could add extra parameter checks, but it seems a little unnecessary